posthog/api/test/test_plugin.py

```diff
@@ -903,15 +903,11 @@ def test_job_trigger(self, patch_trigger_plugin_job, mock_get, mock_reload):
             format="json",
         )
 
-        patch_trigger_plugin_job.assert_has_calls(
-            [
-                mock.call(
-                    name="posthog.tasks.plugins.plugin_job",
-                    queue="posthog-plugins",
-                    args=[self.team.pk, plugin_config_id, "myJob", "stop", {"a": 1}],
-                )
-            ]
-        )
+        self.assertEqual(execute_postgres.call_count, 1)
+        expected_sql = 'SELECT graphile_worker.add_job(\'pluginJob\', \'{"type": "myJob", "payload": {"a": 1, "$operation": "stop"}, '
+        expected_sql += f'"pluginConfigId": {plugin_config_id}, "pluginConfigTeam": {self.team.pk}'
+        expected_sql += "}')"
```

Comment:
Nit: The `+=`s make this a bit harder to read for me. I really like using
```Python
x = (
    "foo "
     "bar"
)
```
for such long strings.